.buildkite/pipelines/es_serverless/emergency_relelease_branch_testing.yml

```diff
@@ -0,0 +1,17 @@
+# https://buildkite.com/elastic/kibana-serverless-emergency-release-branch-testing
+
+## Triggers the artifacts container image build for emergency releases
+agents:
+  queue: kibana-default
+
+notify:
+  - slack: "#kibana-mission-control"
+    if: "build.state == 'passed' || build.state == 'failed' || build.state == 'scheduled'"
+
+steps:
+  - trigger: "kibana-artifacts-container-image"
+    label: ":docker: Build Kibana Artifacts Container Image"
+    build:
+      branch: $BUILDKITE_BRANCH
```

Comment:
If this is not `main` we'll lose some test coverage:

https://github.com/elastic/kibana/blob/c902f90a71b88027b92d339172af8a948a9bb79a/.buildkite/scripts/steps/artifacts/docker_image.sh#L104-L121

If we want test coverage and to not promote the image to development we'll want something like (untested, should probably verify with the pipeline maintainers):
```
diff --git a/.buildkite/scripts/steps/artifacts/docker_image.sh b/.buildkite/scripts/steps/artifacts/docker_image.sh
index 0518e17f652..e45330f27c5 100755
--- a/.buildkite/scripts/steps/artifacts/docker_image.sh
+++ b/.buildkite/scripts/steps/artifacts/docker_image.sh
@@ -116,6 +116,19 @@ steps:
         REMOTE_SERVICE_CONFIG: https://raw.githubusercontent.com/elastic/serverless-gitops/main/gen/gpctl/kibana/dev.yaml
 EOF
 
+else if [["$BUILDKITE_BRANCH" =~ ^deploy-fix@[0-9]{10}$ ]]; then
+  cat << EOF | buildkite-agent pipeline upload
+steps:
+  - label: ":rocket: Run E2E tests to check an update"
+    trigger: service-or-stack-update-e2e-tests
+    build:
+      env:
+        SERVICE: kibana-controller
+        NAMESPACE: kibana-ci
+        IMAGE_NAME: kibana-serverless
+        IMAGE_TAG: "$KIBANA_IMAGE_TAG"
+EOF
+
 else
   echo "Skipping update for untracked branch $BUILDKITE_BRANCH"
 fi
```

Reply:
Good point again. I knew we probably didn't want to promote to dev, since it would be competing with the promotions from the main merges, but if the testing can be done separately, I'll try to add this angle

Reply:
We just discussed this with Thomas: we're going to skip this bit of testing for now, reasons:
- this would be sequentially after the image build, adding +1hr to the lead time
- there's currently a required smoke test by the RM on QA & Staging, which is similar purpose to these tests

We might add this in the future though, once we have more data on emergency releases.